src/main/cpp/t10_max_local.cpp

```diff
@@ -24,5 +24,26 @@
 using namespace std;
 
 int t10_max_local() {
+   int a = 0;
+ int b = 0;
+ int d = 0;
+ int e = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'e' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4349150)